sklearn/metrics/pairwise.py

```diff
@@ -1872,7 +1884,7 @@ def pairwise_distances_chunked(
         ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
         for more details.
 
-    working_memory : int, default=None
+    working_memory : float, default=None
```

Comment:
can working_memory really be float? I think it's always an int.

Reply:
There are test cases such as
https://github.com/scikit-learn/scikit-learn/blob/092caed407f3b60de7677d4353bfe0db20a2faab/sklearn/metrics/tests/test_pairwise.py#L620-L622
so I think float is also accepted.

Reply:
Right, this is in MB, not bytes.